apps/emqx_connector_jwt/src/emqx_connector_jwt.erl

```diff
@@ -132,14 +133,15 @@ do_generate_jwt(#{
     JWT.
 
 -spec store_jwt(jwt_config(), jwt()) -> ok.
-store_jwt(#{resource_id := ResourceId, table := TId}, JWT) ->
-    true = ets:insert(TId, {{ResourceId, jwt}, JWT}),
+store_jwt(#{resource_id := ResourceId} = JWTConfig, JWT) ->
+    Table = maps:get(table, JWTConfig, ?JWT_TABLE),
+    true = ets:insert(Table, {{ResourceId, jwt}, JWT}),
     ?tp(emqx_connector_jwt_token_stored, #{resource_id => ResourceId}),
     ok.
 
 -spec is_about_to_expire(jwt()) -> boolean().
 is_about_to_expire(JWT) ->
     #jose_jwt{fields = #{<<"exp">> := Exp}} = jose_jwt:peek(JWT),
     Now = erlang:system_time(seconds),
-    GraceExp = Exp - 5,
+    GraceExp = Exp - timer:seconds(5),
```

Comment:
This looks incorrect: Now is in seconds, and `timer:seconds` converts seconds to milliseconds.

Reply:
Good catch.  I moved this change from a branch without the fix.  Thanks!